app/scripts/modules/core/src/application/modal/createApplication.modal.controller.js

```diff
@@ -129,5 +129,8 @@ module.exports = angular
         }
         this.createApplicationForTests();
       };
+
+      // Enables setting the attributes as a callback in react components
+      this.setAttribute = (name, value) => (this.application[name] = value);
```

Comment:
doesn't this need to be wrapped in `$applyAsync` for angular to pick it up?

Reply:
I haven't had any problem running locally without that.

Reply:
Let me know if I am missing something, although I suppose it doesn't hurt to add as a safeguard.

Reply:
If it works locally, then 🤷‍♂